### ISSUE TITLE
Helicobacter pylori gastritis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6401]
**Preferred term label:**
Helicobacter pylori gastritis

**Synonyms**
Helicobacter gastritis

**Definition (free text, please give PubMed ID)**
Inflammation of the stomach caused by the bacteria helicobacter pylori. 
DOI: https://doi.org/10.1007/s002920000423 

**Parent term (use hpo.jax.org/app)**
Gastritis

**Comment by pnrobinson** [comment]
I do not think that this qualifies as an "Unusual infection", which would make it a phenotype according to the HPO definition. This is more of a disease, and the MONDO term is https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0006781
